packages/documentation-site/patternfly-docs/content/PatternFly-AI/rapid-prototyping/migrating-to-patternfly.md

```diff
@@ -0,0 +1,74 @@
+### Prompting tips
+When interacting with AI, especially with tools like Cursor, specificity is key. In order to efficiently and successfully request something from an AI tool, there are a few "do's and don'ts" to follow:
+| **Don't** | **Do** |
+| --- | --- |
+| Expect AI to assume your needs. | Be specific and clearly state what you want to achieve. |
+| Let AI follow its own rules. | Instruct AI to follow best practices for React and PatternFly, providing clear definitions and examples for these practices. |
+| Restrict yourself to text-based prompts. | When texts prompts are insufficient, you can share screenshots with AI to provide valuable context. |
+| Reinvent the wheel|  Use previously completed features and functionalities as models for new requests. |
+
+
+## Using AI throughout migration phases
+
+AI serves as a powerful accelerator throughout all steps of the migration process, including learning, planning, and development phases.
+
+### Learning phase
+
+The learning phase of a migration project often requires building project-specific context, such as understanding the architecture, code-base, best practices and diffrences in technologies. You can query AI to help bridge knowledge gaps, set up development environments, and identify relevant code sections.
+
+For example, if your project is written with Angular, AI can help explain Angular-specific patterns (like directives or services) and point to their equivalents in React and PatternFly.
+
+- Query AI to explain things such as Angular-specific patterns (like directives or services) and their React/PatternFly equivalents.
+- Query AI to explain project specific details.
+- Use AI to configure tools like Podman.
+
+### Planning phase
+
+AI aids in estimating migration scope and mapping existing features to latest PatternFly React equivalents:
+
+- Query AI to generate a list of components that need to be migrated.
+- Query AI to identify areas that may be difficult. Examples could include state management and routing.
+
+### Development phase
+
+AI can assist in building features iteratively, particularly in generating:
+
+- Forms
+- Validation logic
+- Reusable components
+
+In our experimentation we converted a simple Angular page to React using PatternFly components. A prompt we used after creating a placeholder page for our experimentation was: "Copy the Angular content credentials page content into the ReactPf5PlaceholderPage.js using React and PatternFly5 using best practices with bias towards how other React and PatternFly files in this project are doing things."
+
+AI was able to successfully recreate page content while adhering to our request: 
+
+**Before:**
+![Before migration - Angular Content Credentials page](./img/content-credentials-before.png)
+
+**After:**
+![After migration - React PatternFly Content Credentials page](./img/content-credentials-after.png)
```

Comment:
```suggestion
**After:**
![After migration - React PatternFly Content Credentials page](./img/content-credentials-after.png)
```

Reply:
Oh, yeah I wasn't thinking to remove it! Was trying to suggest moving the before one above some descriptive text re https://github.com/patternfly/patternfly-org/pull/4807/files#r2395448435https://github.com/patternfly/patternfly-org/pull/4807/files#r2395448435